### ISSUE TITLE
how to use the xml files and where would they reside in the homedir ?

### ISSUE THREAD

**Issue by shirishag75** [mars-sim/mars-sim#211]
I saw 4971, can you share some more details as to where I see it . I am running the headless version so I need to know how to use it. 81635d7

Would this also make it easier for me to say have the default speed to be at 13 instead of 6 ?

`
00-Adir-01:152.048 (Config) MasterClock : Time-ratio : 64x ==> 8192x`

**Comment by mokun** [comment]
I was gonna say the default speed to be at 13 requires a quad core computer with a minimum of 2GHz. Or else the simulation would appear to get stuck.

**Comment by mokun** [comment]
> can you share some more details as to where I see it . I am running the headless version so I need to know how to use it.

Have you looked at any mars-sim xml files in the past ? 

Each xml contains default settings for each aspect of the simulation.

Let's start with `people.xml`. 

I recall asking a while back if you can add more common Indian last name /surname, and the first name into the relevant section. 

See if you can find any government census reference and put the url link there as well. 

Appreciate !

**Comment by shirishag75** [comment]
that is good enough, although I did see https://www.babycenter.in/a25018003/most-popular-indian-girl-names-of-2015 and https://www.babycenter.in/a25018013/most-popular-indian-boy-names-of-2015 which was taken in names of babies.

**Comment by mokun** [comment]
>  can you share some more details as to where I see it . I

The xml files are at `${USER_DIR}/.mars-sim/conf/`.

Similarly, all the saved sim are at `${USER_DIR}/.mars-sim/saved/`.

**Comment by shirishag75** [comment]
right, saw that. Also why just have `last-name value` and not first-name value as well ?

**Comment by mokun** [comment]
We do have a lot `last-name` and `first-name` entries in `people.xml`. But each `first-name` also has a `gender` attribute.

**Comment by bevans2000** [comment]
The website now covers a section of XML files, https://marssim.space/docs/guide/xml-files/
